tslint.json

```diff
@@ -49,7 +49,9 @@
       true,
       "ban-keywords",
       "check-format",
-      "allow-trailing-underscore"
+      "allow-leading-underscore",
```

Comment:
We'll want to allow one or the other, not both, but I'm fine with it for now (I know it's a big one).